.eslintrc.json

```diff
@@ -1,197 +1,92 @@
 {
-  "env": {
-    "es2021": true,
-    "node": true,
-    "jest": true
-  },
-  "extends": [
-    "standard"
-  ],
-  "parser": "@typescript-eslint/parser",
-  "parserOptions": {
-    "ecmaVersion": 12,
-    "sourceType": "module"
-  },
-  "plugins": [
-    "@typescript-eslint"
-  ],
-  "rules": {
-    "indent": 0,
-    "@typescript-eslint/indent": [
-      "error",
-      2
-    ]
-  },
   "overrides": [
     {
-      "files": [
-        "*.graphql"
-      ],
+      "files": ["*.js", "*.ts"],
+      "env": {
+        "es2021": true,
+        "node": true,
+        "jest": true
+      },
+      "extends": ["standard"],
+      "parser": "@typescript-eslint/parser",
+      "parserOptions": {
+        "ecmaVersion": 12,
+        "sourceType": "module"
+      },
+      "plugins": ["@typescript-eslint"],
+      "rules": {
+        "indent": 0,
+        "@typescript-eslint/indent": ["error", 2]
+      }
+    },
+    {
+      "files": ["*.graphql"],
+      "excludedFiles": ["./src/infrastructure/server/graphql/schema.graphql"],
```

Comment:
Without this line, you'll get a false positive ESLint error
```sh
/Users/dimitri/Desktop/typescript.workshop.backend/src/infrastructure/server/graphql/schema.graphql
  2:77  error  Parsing error: [graphql-eslint]: Syntax Error: Unexpected <EOF>
```
This is because we ignore `#import` comments when we lint files, otherwise you'll get errors output twice in `schema.graphql` and in `src/infrastructure/server/graphql/models/**/*.graphql` type files.

P.S. in truth, we are not ignoring  `#import` comments only for `known-fragment-names` and `no-unused-fragments` rules.